app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -87,24 +91,35 @@ public View getView(int position, View convertView, @NonNull ViewGroup parent) {
   @Override
   public View getDropDownView(int position, View convertView, ViewGroup parent) {
     CheckBoxHolder holder;
-    if (convertView == null) {
-      convertView = layoutInflater.inflate(
+    convertView = layoutInflater.inflate(
           R.layout.translation_ab_spinner_item, parent, false);
-      convertView.setTag(new CheckBoxHolder(convertView));
-    }
-    holder = (CheckBoxHolder) convertView.getTag();
+    convertView.setTag(new CheckBoxHolder(convertView));
 
+    holder = (CheckBoxHolder) convertView.getTag();
     holder.checkBoxPosition = position;
     holder.checkBox.setText(translationNames[position]);
     holder.checkBox.setChecked(selectedItems.contains(translations.get(position).filename));
```

Comment:
I got this working by inflating the textview on this condition, is setting visibility on checkboxes necessary for when `position == translationNames.length - 1`
what I have `if (position == translationNames.length - 1){
      convertView = layoutInflater.inflate(R.layout.translation_ab_spinner_more_item, parent, false);
      TextView tv =  (TextView) convertView.findViewById(R.id.textbox);
      tv.setOnClickListener(this.textListener);`